lib/ansible/module_utils/urls.py

```diff
@@ -590,6 +592,8 @@ def get_ca_certs(self):
         paths_checked.append('/etc/ansible')
 
         tmp_fd, tmp_path = tempfile.mkstemp()
+        to_add_fd, to_add_path = tempfile.mkstemp()
```

Comment:
If we do have to keep it separate are we able to remove it when we remove tmp_path around line 728